src/main/java/uk/gov/di/ipv/cri/common/library/service/AuditEventFactory.java

```diff
@@ -38,20 +41,48 @@ <T> AuditEvent<T> create(String eventType, AuditEventContext auditEventContext,
                         eventPrefix + "_" + eventType,
                         issuer);
         if (Objects.nonNull(auditEventContext)) {
-            if (Objects.nonNull(auditEventContext.getPersonIdentity())) {
-                auditEvent.setRestricted(auditEventContext.getPersonIdentity());
-            }
+            Map<String, String> requestHeaders = auditEventContext.getRequestHeaders();
+
+            auditEvent.setRestricted(
+                    createAuditEventResricted(
+                            requestHeaders, auditEventContext.getPersonIdentity()));
+
             auditEvent.setUser(
-                    createAuditEventUser(
-                            auditEventContext.getRequestHeaders(),
-                            auditEventContext.getSessionItem()));
+                    createAuditEventUser(requestHeaders, auditEventContext.getSessionItem()));
         }
         if (Objects.nonNull(extensions)) {
             auditEvent.setExtensions(extensions);
         }
         return auditEvent;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Why has this method been added if it's marked as deprecated? Maybe remove the annotation if it's used and not superseded by anything else?

Reply:
Ah yeah I mistaken it for marking the method as deprecated. This is fine to leave as is.